docs: Clarify setting of restricted view of images (DEV-2961) (#2950)
dasch-swiss/dsp-api: docs/03-endpoints/api-admin/projects.md

Permissions: ProjectAdmin

Request definition:

> 

Author: without empty line, mkdocs doesn't recognize this as bullet point list, and renders it very ugly. See https://docs.dasch.swiss/2023.10.02/DSP-API/03-endpoints/api-admin/projects/#set-restricted-view-settings

```

Operates on the following properties:

- `knora-admin:projectRestrictedViewSize`: the IIIF size value
- `knora-admin:projectRestrictedViewWatermark`: the path to the watermark image. **Currently not used!**

Caveat:

> - This setting only takes effect for users who have restricted view permissions

Author: better now?

> - This setting only takes effect for users who have restricted view permissions

Reviewer: Not really, because the restricted setting affect all users which has no permissions. The "no" is missing in first part. I will give the suggestion.

> Caveat: The restricted view settings only take effect
> for user groups that have restricted view permissions
> on the `knora-api:hasStillImageFileValue` value of StillImageFile resources.
> User groups with view permission or higher will see the image in full resolution.

Reviewer: ```suggestion
Caveat: The restricted view settings only take effect for user groups that have no 
restricted view permissions on the `knora-api:hasStillImageFileValue` value of 
StillImageFile resources. User groups with view permission or higher will see the image 
in full resolution.
```

> Caveat: The restricted view settings only take effect
> for user groups that have restricted view permissions (or no permissions at all)
> on the `knora-api:hasStillImageFileValue` value of StillImageFile resources.
> User groups with view permission or higher will see the image in full resolution.

Reviewer 2: ```suggestion
Note: Restricted view settings only take effect, if a user has "Restricted View" permission on an image.
```

In my opinion, this information is superfluous, because this is the exact definition of what "restricted view permission" means in our system. But if it gets added, I would keep it as concise as possible.

Author: Perhaps it might be obvious for an API-dev, but when I started working on the permissions of the "postcards" project, it wasn't obvious for me. I have accepted your suggestion. Could I get an approve now?

Reviewer 2: Sure! thanks for the contribution.  
(for future reference it may help to re-request a review, then the PR shows up in the "requested reviews" section again, which is what I most regularly check)

Reviewer 2: all good! :)  
my statement was just complicated for "you would have got the approve 10 minutes earlier with a re-request" ^^
